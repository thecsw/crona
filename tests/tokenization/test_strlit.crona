"this is a good string"
"unterminated string
"unterminated string\"
"bad escape char \u"
"bad escape \u and unterminated
"unterminated string with bad escape \e \"
"backslash followed by space: \ is not allowed"