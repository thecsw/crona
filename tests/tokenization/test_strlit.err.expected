FATAL [2,1]: Unterminated string literal ignored
FATAL [3,1]: Unterminated string literal ignored
FATAL [4,1]: String literal with bad escape sequence ignored
FATAL [5,1]: Unterminated string literal  with bad escape sequence ignored
FATAL [6,1]: Unterminated string literal  with bad escape sequence ignored
FATAL [7,1]: String literal with bad escape sequence ignored
